code/datums/components/material_container.dm

```diff
@@ -228,24 +302,25 @@
 	return total_amount_save - total_amount
 
 /// For spawning mineral sheets at a specific location. Used by machines to output sheets.
-/datum/component/material_container/proc/retrieve_sheets(sheet_amt, datum/material/M, target = null)
+/datum/component/material_container/proc/retrieve_sheets(sheet_amt, datum/material/M, atom/target = null)
 	if(!M.sheet_type)
 		return 0 //Add greyscale sheet handling here later
 	if(sheet_amt <= 0)
 		return 0
 
 	if(!target)
-		target = get_turf(parent)
+		var/atom/parent_atom = parent
+		target = parent_atom.drop_location()
 	if(materials[M] < (sheet_amt * MINERAL_MATERIAL_AMOUNT))
 		sheet_amt = round(materials[M] / MINERAL_MATERIAL_AMOUNT)
 	var/count = 0
 	while(sheet_amt > MAX_STACK_SIZE)
-		new M.sheet_type(target, MAX_STACK_SIZE)
+		new M.sheet_type(target, MAX_STACK_SIZE, null, list((M) = MINERAL_MATERIAL_AMOUNT))
```

Comment:
excuse me?

Reply:
This is so that the default material meat stacks don't assimilate the bespoke mat stacks when you break an autolathe. Stacks try to merge themselves into all other stacks of the same type and material in the loc they initialize in. Since meat stacks default to the normal meat material if there any partial stacks of the default meat material on the same turf (as will happen if an autolathe breaks) the bespoke mat stacks merge into the default mat stacks.